unit-tests/py/rspy/test.py

```diff
@@ -680,6 +694,9 @@ class remote:
     """
 
     class Error( RuntimeError ):
+        """
+        Raised when an exception is raised on the remote side, unexpectedly so not the same as CheckFailed
```

Comment:
This comment is not clear. What is the `remote side` and is it enforced that it will raise Error and not some other type (RuntimeError or CheckFailed...)